shell/common/shell_unittests.cc

```diff
@@ -23,6 +24,64 @@
 
 namespace shell {
 
+class TestPlatformView : public PlatformView,
+                         public GPUSurfaceSoftwareDelegate {
+ public:
+  TestPlatformView(PlatformView::Delegate& delegate,
+                   blink::TaskRunners task_runners)
+      : PlatformView(delegate, std::move(task_runners)) {}
+
+ private:
+  // |PlatformView|
+  std::unique_ptr<Surface> CreateRenderingSurface() override {
+    return std::make_unique<GPUSurfaceSoftware>(this);
+  }
+
+  // |GPUSurfaceSoftwareDelegate|
+  virtual sk_sp<SkSurface> AcquireBackingStore(const SkISize& size) override {
+    return SkSurface::MakeRasterN32Premul(size.width(), size.height());
+  }
+
+  // |GPUSurfaceSoftwareDelegate|
+  virtual bool PresentBackingStore(sk_sp<SkSurface> backing_store) override {
+    return true;
+  }
+
+  FML_DISALLOW_COPY_AND_ASSIGN(TestPlatformView);
+};
+
+bool ValidateShell(Shell* shell) {
```

Comment:
Nit: in Skia, we tend to put `static` before these functions (e.g., `static bool ValidateShell`) that are only used in this cc file.

Reply:
Done

Reply:
This is a pattern we follow in the engine too. I was just forgetful :(